common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
```

Comment:
This method follows that of a similar method above. If I changed the way this one template works I'd want to change the other, but I'm thinking that's for another PR. Thoughts?